fault/tester/staged_tester.py

```diff
@@ -422,3 +422,6 @@ def __init__(self, circuit: m.Circuit, clock: m.Clock = None):
 
     def _else(self):
         return ElseTester(self.else_actions, self.circuit, self.clock)
+
+
+StagedTester = Tester
```

Comment:
```python
class StagedTester(Tester):
    pass
```
instead? unless we need `is` relation

Reply:
I think we should eventually deprecate `Tester` and have `StagedTester` be the default name since it's more explicit (compared to interactive Tester) so it's not really meant to be a subclass, rather an alias